src/AutoRest.CSharp/LowLevel/Output/LowLevelClient.cs

```diff
@@ -334,6 +334,61 @@ private IEnumerable<Parameter> GetSubClientFactoryMethodParameters()
             return candidates.OrderBy(c => c.Parameters.Count).FirstOrDefault();
         }
 
+        internal MethodSignatureBase? GetEffectiveCtorWithClientOptions()
```

Comment:
Can we unify it with method above?

Reply:
Yes we could - I was just considering the original copy of this method (`GetEffectiveCtor`) is already a workaround which will be removed in the future therefore to avoid any accident I just duplicated it.
I will unify this with the method above.